libs/tree-sitter-wing/grammar.js

```diff
@@ -320,6 +320,7 @@ module.exports = grammar({
         $.struct_literal,
         $.optional_test,
         $.compiler_dbg_panic,
+        $.compiler_dbg_env,
```

Comment:
note that in this case I'd go for a special statement and not a special expression. I wanted the expression option for `compiler_dbg_panic` so I can cause panics within an expression evaluation. But here I think we can simplify things if we go for a statement which doesn't need to evaluate to any type and can't be used within another expression.

Reply:
Mmmm yeah that makes more sense. Evaluating sub-expressions should never change the environment so there's not really a point nesting it deeply inside expressions